From 54821ed9f986a0023bbc3fa08060e1f2c8efb6b5 Mon Sep 17 00:00:00 2001
From: Chase Maupin <Chase.Maupin@ti.com>
Date: Mon, 24 Feb 2014 13:16:20 -0600
Subject: [PATCH 1/2] ARM: config: omap: Change PREEMPTion to voluntary

* Change the default PREEMPT mode to voluntary which reduces the
  overhead and increases performance for many use cases such as
  UDP networking and cryptography.
* Based on recommendations from Joel Fernandes <joelf@ti.com>

Signed-off-by: Joel Fernandes <joelf@ti.com>
Signed-off-by: Chase Maupin <Chase.Maupin@ti.com>
Signed-off-by: Denys Dmytriyenko <denys@ti.com>
---
 arch/arm/configs/omap2plus_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/configs/omap2plus_defconfig b/arch/arm/configs/omap2plus_defconfig
index 3975b08..a34b45a 100644
--- a/arch/arm/configs/omap2plus_defconfig
+++ b/arch/arm/configs/omap2plus_defconfig
@@ -38,7 +38,7 @@ CONFIG_ARM_ERRATA_411920=y
 CONFIG_PCIE_DRA7XX=y
 CONFIG_SMP=y
 CONFIG_NR_CPUS=2
-CONFIG_PREEMPT=y
+CONFIG_PREEMPT_VOLUNTARY=y
 CONFIG_CMA=y
 CONFIG_ZBOOT_ROM_TEXT=0x0
 CONFIG_ZBOOT_ROM_BSS=0x0
-- 
1.8.3.2

